person.rb

```diff
@@ -0,0 +1,44 @@
+require '/decorate'
+
+# parent class Person.new(age, name, parent_permision)
+class Person < Nameable
+  attr_reader :id, :rentals
+  attr_accessor :name, :age
```

Comment:
- [ ] Same here, Please add Getters and Setters for @rentals instead of getters only.